lib/commands/connect/regions.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+const cli = require('heroku-cli-util');
+
+const prodRegions = ['us', 'eu', 'tokyo', 'virginia', 'oregon', 'frankfurt', 'sydney'];
```

Comment:
Also that puts `dublin` in special snowflake territory (but also covered by `eu`). I wonder how that works with our ability to move forward with preparing new regions using our plugin commands.